<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>372</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>780</width>
    <height>372</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>B2268S Tool v1.00 by RedSec</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
background-color: rgba(0, 0, 0, 70%);
border: 1px solid white; 
color:white;
}

#lineEdit{
background-color: rgba(0, 0, 0, 70%);
border: 1px solid white; 
color:white;
}

#textEdit{
background-color: rgba(0, 0, 0, 70%);
border: 1px solid white; 
color:white;
}
QPushButton {
background-color: rgba(0, 0, 0, 70%);
border: 1px solid white; 
color:white;
padding-top: 3px;
 padding-right: 12px;
 padding-bottom: 3px;
  padding-left: 12px;
}

QPushButton:pressed {
    background-color: rgba(255, 255, 255, 70%);
    border-style: inset;
}

QPushButton:flat {
    border: none; /* no border for a flat push button */
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}

QPushButton:disabled {
	background-color:rgba(255,255,255,70%);
}

QGroupBox {
    border: 2px solid white;
    margin-top: 1ex; /* leave space at the top for the title */
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center; /* position at the top center */
    padding: 0 3px;
}

QMenu::item:selected { /* when user selects item using mouse or keyboard */
    background-color: rgba(255,255,255,70%);
    color:black;
}

QMenu {
    background-color: rgba(0,0,0,10%); /* sets background of the menu */
}

QMenu::item {
    /* sets background of menu item. set this to something non-transparent
        if you want menu color and menu item color to be different */
    background-color: transparent;
}

QMenuBar {
    background-color: rgba(0,0,0,70%);
   color:yellow;
font-weight: bold;
}

QMenuBar::item {
    spacing: 3px; /* spacing between menu bar items */
    padding: 1px 4px;
    background: transparent;
}

QMenuBar::item:selected { /* when selected using mouse or keyboard */
    background: rgba(255,255,255,70%);
	color:yellow;
font-weight: bold;
}

QMenuBar::item:pressed {
    background:  rgba(255,255,255,70%);
   color:yellow;
font-weight: bold;
}

QProgressBar {
    border-color:white;
	color:white;
	text-align: center;
}

QProgressBar::chunk {
    background-color: rgba(255,255,255,70%);
    width: 20px;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QTextEdit" name="textEdit">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>CPE B2268S</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Unlock Admin</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_4">
             <property name="text">
              <string>Ping B2268S</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>E5172 Outdoor Unit</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QPushButton" name="pushButton_2">
             <property name="text">
              <string>Reboot ODU</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_3">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Reset ODU Firmware</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_5">
             <property name="text">
              <string>Ping E5172</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>DHCP Pool Size</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0" colspan="2">
            <widget class="QLineEdit" name="lineEdit">
             <property name="placeholderText">
              <string>type dhcp pool size here</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="pushButton_6">
             <property name="text">
              <string>Save</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="pushButton_7">
             <property name="text">
              <string>Show Hosts</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="QWebView" name="webView">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="url">
       <url>
        <string>qrc:/others/asd.html</string>
       </url>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>780</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>B2268S Tool</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionUpdates"/>
    <addaction name="actionAbout_B2268S_Tool"/>
    <addaction name="actionAbout_RedSec"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>View Help</string>
   </property>
  </action>
  <action name="actionAbout_B2268S_Tool">
   <property name="text">
    <string>About B2268S Tool</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionAbout_RedSec">
   <property name="text">
    <string>About Android 20</string>
   </property>
  </action>
  <action name="actionUpdates">
   <property name="text">
    <string>View Updates</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
